test/module/irohad/common/validators_config.hpp

```diff
@@ -19,6 +20,11 @@ namespace iroha {
             std::make_shared<shared_model::validation::ValidatorsConfig>(
                 getTestsMaxBatchSize()));
 
+    static const std::shared_ptr<shared_model::validation::ValidatorsConfig>
+        kTestsValidatorsCustomConfig(
+            std::make_shared<shared_model::validation::ValidatorsConfig>(
+                getTestsMaxBatchSize(), std::make_shared<iroha::ametsuchi::SettingsStub>()));
```

Comment:
```suggestion
                getTestsMaxBatchSize(), kTestSettings));
```
and above
```c++
static const std::shared_ptr<Settings> kTestSettings = std::make_shared<iroha::ametsuchi::SettingsStub>();
```
to be reused in tests

Reply:
fixed